chromadb/segment/impl/manager/distributed.py

```diff
@@ -142,14 +142,14 @@ def _segment(type: SegmentType, scope: SegmentScope, collection: Collection) ->
     # For the segment types with python implementations, we can propagate metadata
     if type in SEGMENT_TYPE_IMPLS:
         cls = get_class(SEGMENT_TYPE_IMPLS[type], SegmentImplementation)
-        collection_metadata = collection.get("metadata", None)
+        collection_metadata = collection.metadata
```

Comment:
why this change?

Reply:
Collection model is no longer a `TypedDict`, so while it does have a `__getitem__` allowing us to do e.g. `collection['metadata']`, it no longer has a `get` impl. 

Acccessing members directly this way preserves type information too.